webapp/client/src/components/FormFieldDropDown.js

```diff
@@ -0,0 +1,97 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import FormFieldScaffolding from "./FormFieldScaffolding";
+import FieldLabel from "./FieldLabel";
+import selectIcon from "../assets/icons/select_icon.svg";
+
+const DropDown = styled.div`
+  & .steuerlotse-select {
+    border: 2px solid var(--border-color);
+    border-radius: 0;
+    background-image: url(${selectIcon});
+    background-repeat: no-repeat;
+    background-size: 0.75rem;
+    min-height: 55px;
+  }
+
+  & .steuerlotse-select:focus {
+    border: 2px solid var(--focus-border-color);
+    box-shadow: 0 0 0 2px var(--focus-color);
+  }
+`;
+
+function FormFieldDropDown({
+  fieldName,
+  fieldId,
+  options,
+  defaultOption,
+  selectedValue,
+  required,
+  label,
+  details,
+  errors,
+  onChangeHandler,
+}) {
+  return (
+    <FormFieldScaffolding
+      {...{
+        fieldName,
+        errors,
+        cols: "6",
+      }}
+      hideLabel
+      render={() => (
+        <DropDown>
+          <FieldLabel {...{ label, fieldId, details }} />
+          <select
+            id={fieldId}
+            className="custom-select steuerlotse-select"
+            name={fieldId}
+            defaultValue={selectedValue}
+            required={required}
+            autoFocus={Boolean(errors.length)}
+            onBlur={onChangeHandler}
+            onChange={onChangeHandler}
+          >
+            {defaultOption && (
+              <option value="" key={fieldId + defaultOption}>
+                {defaultOption}
+              </option>
+            )}
```

Comment:
Shouldn't the default option just be set in the backend to the wtforms field? Right now it is in some cases (e.g. Bundesland) while for the tax offices it's set in JavaScript.

Reply:
I think I do not really get what you mean? We need to set the defaultOption in the React component if it is coming from the backend or if it is always the same for all components.

Reply:
Okay, I think I understood you correctly then :) I think there are two aspects of that problem:
1. We need to tell the frontend that we have a default option. So far, that could be done by the backend because we used WTForms for the creation of the HTML. When using this React component it needs to take care of setting the default option. That is why this part is here.
2. The integration aspect. We could change it here to not be adaptable and always say "Bitte auswählen" or at least be the default option. I think that this makes a lot of sense and we should change that. However, I would leave that aspect to the integration PR.

I hope that this makes it a little bit clearer. Otherwise, let's talk ;)

Reply:
Oh god, I'm a bit afraid to ask..but why?
> We need to tell the frontend that we have a default option.
Isn't that just another option? Or is it special to the frontend in any way?

Reply:
I think this didn't become clear in my comments so far: I'm not at all convinced we *should* set it in the backend. Just that we should do it consistently.

Reply:
I'm with Frede here. Picking what default option to show feels like a presentation thing, not a data thing, and therefore belongs into the frontend.

Reply:
Actually, maybe that's what you're already suggesting @JulianRoesner? (Definitely something for the integration PR though, so let's maybe have that conversation then…)

Reply:
> Actually, maybe that's what you're already suggesting @JulianRoesner? 

Yes, that's what I meant.